Performing C SOURCE FILE Test CMAKE_HAVE_LIBC_PTHREAD failed with the following output:
Change Dir: /home/douglas/desktop/TCC/xbow440/build/CMakeFiles/CMakeScratch/TryCompile-y3D8SU

Run Build Command(s):/usr/bin/make -f Makefile cmTC_ee42f/fast && /usr/bin/make  -f CMakeFiles/cmTC_ee42f.dir/build.make CMakeFiles/cmTC_ee42f.dir/build
make[1]: Entrando no diretório '/home/douglas/desktop/TCC/xbow440/build/CMakeFiles/CMakeScratch/TryCompile-y3D8SU'
Building C object CMakeFiles/cmTC_ee42f.dir/src.c.o
/usr/bin/cc -DCMAKE_HAVE_LIBC_PTHREAD   -o CMakeFiles/cmTC_ee42f.dir/src.c.o -c /home/douglas/desktop/TCC/xbow440/build/CMakeFiles/CMakeScratch/TryCompile-y3D8SU/src.c
Linking C executable cmTC_ee42f
/home/douglas/.local/lib/python3.10/site-packages/cmake/data/bin/cmake -E cmake_link_script CMakeFiles/cmTC_ee42f.dir/link.txt --verbose=1
/usr/bin/cc -rdynamic CMakeFiles/cmTC_ee42f.dir/src.c.o -o cmTC_ee42f 
/usr/bin/ld: CMakeFiles/cmTC_ee42f.dir/src.c.o: na função "main":
src.c:(.text+0x46): referência não definida para "pthread_create"
/usr/bin/ld: src.c:(.text+0x52): referência não definida para "pthread_detach"
/usr/bin/ld: src.c:(.text+0x5e): referência não definida para "pthread_cancel"
/usr/bin/ld: src.c:(.text+0x6f): referência não definida para "pthread_join"
collect2: error: ld returned 1 exit status
make[1]: *** [CMakeFiles/cmTC_ee42f.dir/build.make:99: cmTC_ee42f] Erro 1
make[1]: Saindo do diretório '/home/douglas/desktop/TCC/xbow440/build/CMakeFiles/CMakeScratch/TryCompile-y3D8SU'
make: *** [Makefile:127: cmTC_ee42f/fast] Erro 2


Source file was:
#include <pthread.h>

static void* test_func(void* data)
{
  return data;
}

int main(void)
{
  pthread_t thread;
  pthread_create(&thread, NULL, test_func, NULL);
  pthread_detach(thread);
  pthread_cancel(thread);
  pthread_join(thread, NULL);
  pthread_atfork(NULL, NULL, NULL);
  pthread_exit(NULL);

  return 0;
}


Determining if the function pthread_create exists in the pthreads failed with the following output:
Change Dir: /home/douglas/desktop/TCC/xbow440/build/CMakeFiles/CMakeScratch/TryCompile-aSG3eT

Run Build Command(s):/usr/bin/make -f Makefile cmTC_f6f49/fast && /usr/bin/make  -f CMakeFiles/cmTC_f6f49.dir/build.make CMakeFiles/cmTC_f6f49.dir/build
make[1]: Entrando no diretório '/home/douglas/desktop/TCC/xbow440/build/CMakeFiles/CMakeScratch/TryCompile-aSG3eT'
Building C object CMakeFiles/cmTC_f6f49.dir/CheckFunctionExists.c.o
/usr/bin/cc   -DCHECK_FUNCTION_EXISTS=pthread_create -o CMakeFiles/cmTC_f6f49.dir/CheckFunctionExists.c.o -c /home/douglas/desktop/TCC/xbow440/build/CMakeFiles/CMakeScratch/TryCompile-aSG3eT/CheckFunctionExists.c
Linking C executable cmTC_f6f49
/home/douglas/.local/lib/python3.10/site-packages/cmake/data/bin/cmake -E cmake_link_script CMakeFiles/cmTC_f6f49.dir/link.txt --verbose=1
/usr/bin/cc  -DCHECK_FUNCTION_EXISTS=pthread_create -rdynamic CMakeFiles/cmTC_f6f49.dir/CheckFunctionExists.c.o -o cmTC_f6f49  -lpthreads 
/usr/bin/ld: não foi possível localizar -lpthreads
collect2: error: ld returned 1 exit status
make[1]: *** [CMakeFiles/cmTC_f6f49.dir/build.make:99: cmTC_f6f49] Erro 1
make[1]: Saindo do diretório '/home/douglas/desktop/TCC/xbow440/build/CMakeFiles/CMakeScratch/TryCompile-aSG3eT'
make: *** [Makefile:127: cmTC_f6f49/fast] Erro 2



